src/__tests__/price_guidance_integration.test.ts

```diff
@@ -0,0 +1,112 @@
+jest.mock("../utils/getPriceGuidance")
+import { getPriceGuidance } from "../utils/getPriceGuidance"
+;(getPriceGuidance as any).mockResolvedValue(1000)
```

Comment:
Hmm, would adding type info to the function signature of `getPriceGuidance` make this unnecessary?

(Not sure myself, I've def been confused by TS + Jest + mocking before)